src/config.py

```diff
@@ -130,8 +130,8 @@ class Status:
     class Substrate:
         """Substrate related constants."""
 
-        VM = "vm"
         K8S = "k8s"
+        VM = "vm"
```

Comment:
This was done by the linter?

Reply:
I believe I did it while playing with somethings.

Reply:
I'll revert it